client/my-sites/email/inbox-management/mailbox-selection-list/style.scss

```
@@ -0,0 +1,126 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+body {
+	background-color: #fff;
+}
+
+.mailbox-selection-list {
+
+	.mailbox-selection-list__header {
+		.formatted-header__title {
+			font-size: $font-title-large;
+			line-height: $font-headline-medium;
+
+			@include break-large {
+				font-size: $font-headline-medium;
+			}
+		}
+
+		.formatted-header__subtitle {
+			font-size: $font-body;
+			line-height: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__status {
+		line-height: 24px;
+		font-size: $font-title-small;
+		position: fixed;
+		top: 50%;
+		left: 50%;
+		transform: translate( -50%, -50% );
+		display: flex;
+		align-items: center;
+		justify-content: center;
+
+		.gridicon {
+			margin-right: 14px;
+			width: 40px;
+			height: 40px;
+		}
+
+		.gridicons-cross-circle {
+			color: #ff0000;
+		}
+
+		@include break-large {
+			transform: none;
+			font-size: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__item {
+		width: 343px;
+		box-shadow: none;
+		border-width: 1px 0 0;
+		border-color: #e7e9ea;
+		border-style: solid;
+		margin-bottom: 1px;
+		display: flex;
+		align-items: center;
+
+		@include break-large {
+			width: 660px;
+		}
+
+		.gridicons-external {
+			width: 20px;
+		}
+
+		&:first-of-type {
+			margin-top: 60px;
+			border-top-width: 0;
+		}
+
+		.mailbox-selection-list__domain h2 {
+			color: var( --p2-color-text );
+			font-family: SF UI Text, SFUIText-Regular, Helvetica Neue, HelveticaNeue, Helvetica, Arial, sans-serif;
```

As above RE the font-family needing to be `$sans`.
```suggestion
			font-family: $sans;
```